src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -482,6 +489,19 @@ export class CourseUpdateComponent implements OnInit {
     get isValidConfiguration(): boolean {
         return this.isValidDate;
     }
+
+    /**
+     * Deletes the course icon
+     * @param courseId id the course that will be deleted
+     */
+    deleteCourseIcon() {
+        this.course.courseIcon = undefined;
+        this.courseImageFile = undefined;
+        this.courseImageFileName = this.course.courseIcon!;
+        this.croppedImage = '';
```

Comment:
Why do you set this to an empty string instead of undefined as the rest?

Reply:
croppedImage is used here: [course-update.component.ts#L274](https://github.com/ls1intum/Artemis/blob/52869217a3c555e19eeda0d69607a771197e9fc9/src/main/webapp/app/course/manage/course-update.component.ts#L274), setting it to undefined will cause it to break. Also I set it to empty string, to make it consistent with the other assignment (e.g: [course-update.component.ts#L188](https://github.com/ls1intum/Artemis/blob/52869217a3c555e19eeda0d69607a771197e9fc9/src/main/webapp/app/course/manage/course-update.component.ts#L188)).